raiden-ts/src/path/types.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import * as t from 'io-ts';
+import { Address } from '../utils/types';
+
+export const PathResults = t.readonly(
+  t.intersection([
+    t.type({
+      result: t.readonlyArray(
+        t.readonly(
+          t.type({
+            path: t.readonlyArray(Address),
+            estimated_fee: t.number,
+          }),
+        ),
+      ),
+    }),
+    t.partial({
+      feedback_token: t.string,
```

Comment:
According to the PFS docs we somehow need to provide feedback about the requested routes with this `feedback_token`, when doing a payment.

Although the docs don't go into detail on how this should be done, do we still need to do something from the SDK side with it?

Reply:
According to @palango it's completely optional and the feedback data is not actively used at the moment

Reply:
I have created the follow up ticket #358